namui/src/namui/manager/web/mouse_manager.rs

```diff
@@ -140,6 +140,7 @@ impl MouseCursor {
             Self::Grab => "grab",
             Self::Move => "move",
             Self::Pointer => "pointer",
+            MouseCursor::Custom(_) => "none",
```

Comment:
@bigfoodK I already did it
https://github.com/NamseEnt/namseent/pull/236#issuecomment-1166344180

Reply:
Aaaah. sorry